src/views/wait-for-others-to-sign.njk

```diff
@@ -11,30 +11,17 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
         <div class="interruption-card">
-          <h1 id="blue-interruption-card" class="govuk-heading-xl govuk-!-margin-bottom-1">You must ask the other directors or members to sign the application</h1>
+          <h1 id="blue-interruption-card" class="govuk-heading-xl govuk-!-margin-bottom-1">The directors or members must sign the application before you can submit it</h1>
         </div>
       </div>
     </div>
 
     <div class="govuk-grid-row govuk-!-margin-top-4">
       <div class="govuk-grid-column-two-thirds">
-
-        <h2 class="govuk-heading-s">Before submitting, you must now:</h2>
-        <ul class="govuk-list govuk-list--bullet">
-          <li>email the other directors or members or those signing for them to ask them to sign the application</li>
-          <li>give them the link to the Apply to strike off or dissolve a company service: <a href="#" class="govuk-link">http://guidedfiling.companieshouse.gov.uk/start?t=DS01</a></li>
-        </ul>
-
-        <h2 class="govuk-heading-s">Those signing must:</h2>
-        <ul class="govuk-list govuk-list--bullet">
-          <li>select the link and start to go through the service</li>
-          <li>sign in to or register for a Companies House account using the email address that you have given us in this service</li>
-          <li>have the company's authentication code in order to sign</li>
-        </ul>
-
-        <p class="govuk-body">When all directors or members have signed you will be able to pay for and submit the application.</p>
-        <p class="govuk-body">You can now exit this page</p>
-
+        <h2 class="govuk-heading-s">The directors or members must sign the application before you can submit it</h2>
```

Comment:
Confirmed with Jayti that this needs to be removed, otherwise we have a duplicated text (with the blue box).